### PR TITLE
Nicer pusher configuration

### PR DESCRIPTION
**Pull request by bitamar**

### CODE REVIEW

**Comment by AronNovak** [comment]
needs a reroll

**Comment by bitamar** [comment]
@AronNovak - Ready for review again

File: server/scripts/reset [hunk 1]
```diff
@@ -147,6 +147,9 @@ create_sites_default_files_directory
 # Enable the development modules.
 enable_development_modules
 
+# Update settings.php.
+update_settings
```
Comment (AronNovak) [inline server/scripts/reset]: Checking it quickly, won't we end up with several Pusher config sections in `settings.php` after executing `reset` again and again?

Comment (bitamar) [inline server/scripts/reset]: I checked now. It works ok, because the reset is deleting `settings.php`

**Pull request merged**